content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -307,11 +310,14 @@ stages:
 
 ```yaml
 # dvc.lock
+schema: '2.0'
 stages:
   build@uk:
     cmd: python train.py 'uk' 3 10
     outs:
-      - model-uk.hdfs
+      - path: model-uk.hdfs
+        md5: 17b3d1efc339b416c4b5615b1ce1b97e
+        size: 2712300
```

Comment:
Actually, they are OK in the latter example but I'll remove it from here to simplify. I'll make a follow-up PR to explain the `size` (and `nfiles`?) fields in dvc.lock.

```suggestion
```